### ISSUE TITLE
Rendering error in main inventory menu

### ISSUE THREAD

**Issue by JakeThatDog** [FTBTeam/FTB-Modpack-Issues#6118]
### Modpack

FTB Evolution

### Modpack version

1.1.1

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

When accessing the main inventory, JEI is completely unrendered, but still functional. On top of this, item names on hover are also not visible. These issues only affect the main menu, working as expected in every other inventory menu. The strength buff icon also seems to be missing, but as potion effects aren't visible in other menus by default I cannot tell if that is a related issue.

### Steps to reproduce

1. Open Inventory
2. Hover on any item

### Expected behaviour

Visible jei menu and search bar in inventory

### Screenshots

![2024-11-08_08 02 09](https://github.com/user-attachments/assets/1a0229da-5b8a-4805-b866-0c26c4171579)
![2024-11-08_08 03 18](https://github.com/user-attachments/assets/d19f7d94-b3d8-4cf3-93d5-f91a29757566)

**Comment by Temil2006** [comment]
check your equipted items, remove them piece by piece to see when it shows. My Son had the same issue, it was caused by the "Luck" applied to the chest piece. Not sure if it's only Luck that casues it or others as well.

One temp work around is to click the "curios" circle button on top left char picture. it doesn't have the issue within that menu, but you still have access to inventory.

**Comment by JakeThatDog** [comment]
The issue still persisted after removing all my equipped items and cleaning out my inventory. Curios menu does work fine as a temporary fix, ill use that instead.

**Comment by alfredggttv** [comment]
Is this on a server by chance?

**Comment by Darkfiend009** [comment]
I can confirm it occurs on a server as well

**Comment by alfredggttv** [comment]
I'm currently trying to recreate the bug, and haven't run into the issue yet myself. Other players are also reporting it as well. I'll keep investigating!

**Comment by dcm2610** [comment]
Can confirm, issue present on both singleplayer and servers

**Comment by KittenIsLive** [comment]
This seems to happen to me when I have more than 1 potion effect active at once. I'm using the potion pylon, having just one of my potion cards in allows me to access JEI normally, but any after that disables it.

**Comment by JakeThatDog** [comment]
My resolution is 1920x1080 and I run the game at UI scale 2

**Comment by JakeThatDog** [comment]
I can confirm that the issue persisted into the newest update

**Comment by bbqx** [comment]
Same issues, Started after I hit 3 buffs from the Spice of life mod. 

The upper left of the screen has the FTB shortcuts, I clicked the gear and the 2nd to last option in the list closed all the icon in the upper left. It stopped the issue, but now I want the icons to comeback so I can use the trashcan. A bit off topic, but if anyone knows id like to know how to bring the icons in the upper left back. 

Still I hope its helpful to know what closing that upper left menu had an effect on the issue. Maybe a hint to lead to what's causing it?

**Comment by JakeThatDog** [comment]
> Same issues, Started after I hit 3 buffs from the Spice of life mod.
> 
> The upper left of the screen has the FTB shortcuts, I clicked the gear and the 2nd to last option in the list closed all the icon in the upper left. It stopped the issue, but now I want the icons to comeback so I can use the trashcan. A bit off topic, but if anyone knows id like to know how to bring the icons in the upper left back.
> 
> Still I hope its helpful to know that closing that upper left menu had an effect on the issue. Maybe a hint to lead to what's causing it?

Re-enable the buttons, right click them, it pulls up a menu to remove specific ones. I actually managed to fix my issue when i removed the creative-locked buttons and Team menu, so I have to presume the entire issue overall is linked to too many things being rendered at once. Thanks for the assistance!

**Comment by dcm2610** [comment]
I think this issue needs be reopened due to the fix just being a workaround that ideally needs to be addressed by the modpack developers

**Comment by JakeThatDog** [comment]
I certainly could reopen it so we get a more concrete solution, but I don't know for sure what the developers would do to fix it otherwise. seems like a issue with JEI itself

**Comment by Zegveld** [comment]
Did some checking and it looks like an issue caused by the `FTB Library Client`.
![image](https://github.com/user-attachments/assets/3c77d2ce-1f43-4b15-bd4b-5dc30a9d29f0)

Workaround:
Disabling all these toggle buttons: `gamemode`, `rain`, `night`, `day`.
Fixes the issue.
![image](https://github.com/user-attachments/assets/d54bf04c-39b2-4849-92e8-1e67cb8ed133)

Additional information:
Also when going into creative mode (used `open to lan`, to enable cheats, then changed world to creative)
The buttons are now visible, and the issue is also gone.

![image](https://github.com/user-attachments/assets/12c13205-a3a3-4955-9a02-e119b9656359)

**Comment by JakeThatDog** [comment]
Yeah, that's pretty much what I did. Honestly i'm not leaving this up anymore it's pretty done and dusted

**Comment by Zegveld** [comment]
My suggestion would still be to have these buttons removed by default, instead of having everyone remove them manually after installation.

State at clean installation:
![image](https://github.com/user-attachments/assets/78cbef78-6ef9-4ac1-90e0-6344cf910007)

That way people won't run into the issue anymore, even though it might still exist.

**Comment by Temil2006** [comment]
what i did to fix / work around this issue was .. install "EMI" on the clients as an additional mod not part of the FTB pack. EMI acually fixes 2 issues.. 
first being the JEI menu not showing and the tooltips not showing.
second being missing recipes from JEI ..for example, "Fluid Hopper" is a recipe you can find by sifted through the middle crafting areas looking at all recipes that use "white chalk" .. but when you search for it via the search option on the bottom, the item never shows. I found there are multiple items missing from JEI and also multiple tabs / acquisition methods .. but they show in EMI ..for example, JEI for Emerald shows 36 tabs of recipes, while EMI shows 38 tabs, so 2 extra machine uses. 

Just and added work around is all.

**Comment by ianwijma** [comment]
My partner ran into the same issue, thanks to @Zegveld we managed to figure it out. Thanks lots!

Opping her "Fixed" the issue.

**Comment by alexmandrikdev** [comment]
> what i did to fix / work around this issue was .. install "EMI" on the clients as an additional mod not part of the FTB pack. EMI acually fixes 2 issues.. first being the JEI menu not showing and the tooltips not showing. second being missing recipes from JEI ..for example, "Fluid Hopper" is a recipe you can find by sifted through the middle crafting areas looking at all recipes that use "white chalk" .. but when you search for it via the search option on the bottom, the item never shows. I found there are multiple items missing from JEI and also multiple tabs / acquisition methods .. but they show in EMI ..for example, JEI for Emerald shows 36 tabs of recipes, while EMI shows 38 tabs, so 2 extra machine uses.
> 
> Just and added work around is all.

I’ve also installed EMI and encountered the same issue when the character has more than one effect applied.
![image](https://github.com/user-attachments/assets/f82b2768-e3dd-4881-8ade-8d8afe1c1d7e)

**Comment by KittenIsLive** [comment]
@alexmandrikdev What modpack version are you playing on? I was pretty sure this issue was fixed in pack version 1.4.0, are you currently playing on the latest 1.6.0?

**Comment by alexmandrikdev** [comment]
@KittenIsLive, I'm using version 1.3.0 because that's the latest one available on CurseForge. I had no idea there were newer versions! Thank you so much for pointing that out. I'll download the pack through the FTB launcher and continue my journey from there.

![image](https://github.com/user-attachments/assets/df4893fa-2854-41a3-995d-6a3b673eca38)

**Comment by Gaz492** [comment]
Just wanted to check now that 1.6.0 is available on curseforge is this still happening to any of you?
